localization/french/culture/wc_culture_titles_l_french.yml

```
@@ -205,6 +215,12 @@
  drottning_female_kvaldir_duke:0 "Drottning"
  drottlag_kvaldir_duchy:0 "Drottlag"
 
+ ### Lucardir Special ###
+ # Feudal
+ blood_thane_male_lucardir_count:0 "Thane de sang"
+ blood_thane_female_lucardir_count:0 "Thane de sang"
+ blood_thanedom_lucardir_county:0 "Terre du Thane de Sang"
```

```suggestion
 blood_thanedom_lucardir_county:0 "Châtellerie de sang"
```
C'est trop long je trouve, je propose ça pour remplacer.